package.json

```diff
@@ -52,9 +51,10 @@
     "install:backend": "poetry install --no-root",
     "install:frontend": "cd frontend && yarn",
     "lint:frontend": "cd frontend && yarn lint",
-    "start": "electron .",
+    "start": "dotenv -e .env -- yarn electron .",
```

Comment:
```suggestion
    "start": "yarn electron .",
    "dev": "dotenv -e .env -- yarn start"
```

Reply:
seperating these as I'm not sure how it will function in production mode for now; in production the envs are passed as expected. it's mostly dev where there are discrepancies around env variables